mm2src/mm2_test_helpers/src/for_tests.rs

```diff
@@ -1664,14 +1665,37 @@ impl MarketMakerIt {
         .map_err(|e| ERRL!("{:?}", e))
     }
 
-    /// Currently, we cannot wait for the `Completed IAmrelay handling for peer` log entry on WASM node,
-    /// because the P2P module logs to a global logger and doesn't log to the dashboard.
+    /// Check if the node is connected to at least one seednode,
+    /// the rpc is used instead of checking the log for DEBUG messages (to opt out p2p debug logging)
     #[cfg(not(target_arch = "wasm32"))]
     pub async fn check_seednodes(&mut self) -> Result<(), String> {
-        // wait for at least 1 node to be added to relay mesh
-        self.wait_for_log(22., |log| log.contains("Completed IAmrelay handling for peer"))
+        let timeout_sec = 22.;
+        let start = now_float();
+        loop {
+            let res = self.rpc(&json!({
+                "userpass": self.userpass,
+                "method": "get_directly_connected_peers",
+            }))
             .await
-            .map_err(|e| ERRL!("{}", e))
+            .unwrap();
+            if res.0.is_success() {       
+                let res_value = serde_json::from_str::<Json>(&res.1).unwrap();
+                if let Some(peers) = res_value["result"].as_object() {
+                    if peers.len() > 0 {
+                        return Ok(());
+                    }
+                }
+            }
+            if now_float() - start > timeout_sec {
+                return ERR!("Timeout expired waiting for connected peers");
+            }
+            if let Some(ref mut pc) = self.pc {
+                if !pc.running() {
+                    return ERR!("MM process terminated prematurely at: {:?}.", self.folder);
+                }
+            }
```

Comment:
didn't understand why we had this, but looks like it's some logic found in `wait_for_log` that was used before.

i think the fact that the rpc replied to us is enough to determine that mm2 is alive, so such a check isn't really needed.